charts/gatekeeper/crds/constrainttemplate-customresourcedefinition.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.8.0
+    controller-gen.kubebuilder.io/version: v0.5.0
```

Comment:
This looks weird as we are going backwards. Looks like this was accidentally bumped by https://github.com/open-policy-agent/gatekeeper/pull/1857/files outside of cutting any releases. v3.8.0 release should bump this back to `0.8.0`. We should be careful to not touch things in charts/ outside of cutting releases. @willbeason @maxsmythe